mopidy/core/playback.py

```diff
@@ -541,3 +541,25 @@ def _trigger_seeked(self, time_position):
         # TODO: Trigger this from audio events?
         logger.debug('Triggering seeked event')
         listener.CoreListener.send('seeked', time_position=time_position)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        return models.PlaybackState(
+            tl_track=self.get_current_tl_track(),
+            position=self.get_time_position(),
+            state=self.get_state())
+
+    def _restore_state(self, state, coverage):
+        """Internal method for :class:`mopidy.Core`."""
+        if state:
+            if not isinstance(state, models.PlaybackState):
+                raise TypeError('Expect an argument of type "PlaybackState"')
+            new_state = ''
```

Comment:
`new_state = None` - I also suspect this logic could be refactored a bit, but doesn't look incorrect so leaving it for now is fine.